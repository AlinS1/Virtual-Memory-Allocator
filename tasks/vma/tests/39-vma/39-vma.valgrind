==5697== Memcheck, a memory error detector
==5697== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==5697== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==5697== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==5697== Parent PID: 5696
==5697== 
==5697== 
==5697== HEAP SUMMARY:
==5697==     in use at exit: 0 bytes in 0 blocks
==5697==   total heap usage: 3,100 allocs, 3,100 frees, 109,168 bytes allocated
==5697== 
==5697== All heap blocks were freed -- no leaks are possible
==5697== 
==5697== For lists of detected and suppressed errors, rerun with: -s
==5697== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
