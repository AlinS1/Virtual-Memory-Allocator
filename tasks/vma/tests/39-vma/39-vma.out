Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
The allocated address is outside the size of arena
The allocated address is outside the size of arena
Invalid address for free.
Invalid address for free.
Invalid address for free.
The allocated address is outside the size of arena
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
The allocated address is outside the size of arena
The allocated address is outside the size of arena
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
This zone was already allocated.
Total memory: 0x204C51 bytes
Free memory: 0x204C3F bytes
Number of allocated blocks: 3
Number of allocated miniblocks: 3

Block 1 beg